src/network/network_gui.cpp

```diff
@@ -750,9 +750,8 @@ class NetworkGameWindow : public Window {
 				break;
 
 			case WID_NG_ADD: // Add a server
-				SetDParamStr(0, _settings_client.network.connect_to_ip);
 				ShowQueryString(
-					STR_JUST_RAW_STRING,
+					GetString(STR_JUST_RAW_STRING, _settings_client.network.connect_to_ip),
```

Comment:
With `STR_JUST_INT` I was thinking maybe use `fmt::format("{}", param)` as that'd be simpler, but I could imagine that being influenced by some custom formatting or so in the future. So I could see merit in doing it with `GetString`.

With `STR_JUST_RAW_STRING` I am really wondering what the merit is to use `GetString`. In my opinion that only exists to be able to inject a raw string into the formatting. Now you change `ShowQueryString` to accept raw  strings, why not pass it directly instead of going through `GetString`?

Reply:
Hmm, indeed, there was some reason to process some strings through `STR_JUST_RAW_STRING`, but this is likely not one of them.